documentation/public/user-guide/query-language.md

```diff
@@ -0,0 +1,109 @@
+# The Tree Query Language
```

Comment:
nit: I think there is a mismatch between:

- page title: `The Tree Query Language`
- page name/URL: `query-language`
- NAV name: `Tree Query Language`

I suggest standardizing on `Tree Query Language`/`tree-query-language` everywhere.